### DISCUSSION TITLE
Enterprise Rollout - prerequisites

### DISCUSSION THREAD

**Discussion by pieewiee** [jgraph/drawio-desktop#1729] [Q&A]
Hello,

has anybody conducted an enterprise rollout for draw.io. We are considering rolling out the software on 50k Win10 & Win11 machines.

**Questions:**
- license that we need to obtain
- License Limimations
- Auto Update recommendations
- Feature customization to hide feature which are not for the ordinary end users (Plugins, Developer mode)

**Comment by davidjgraph** [comment]
> license that we need to obtain + License Limimations

It's [Apache 2.0 licensed](https://github.com/jgraph/drawio-desktop/blob/dev/LICENSE).

> Auto Update recommendations

I'd leave the defaults and let it update when users are prompted. Much of the updating relates to security fixes in updated chrome releases.

> Feature customization to hide feature which are not for the ordinary end users (Plugins, Developer mode)

It's all disabled for the default install.

**Comment by pieewiee** [comment]
would it be possible to disable the "new version prompt"?

**Reply by davidjgraph** [reply]
Everything is possible :). Add as feature request please.

**Reply by whooze** [reply]
I'd say that adding this to  C:\Users\<username>\AppData\Roaming\draw.io\config.json makes draw.io to not look for updates
`{
	"dontCheckUpdates": true
}`

Would be nice to be possible to set that setting out of AppData like in c:\program files\draw.io\ so it can't be changed by the users
